CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 
 ##### Commits
 
+- [[`e12c916499`](https://github.com/postlight/mercury-parser/commit/e12c916499)] - **feat**: ability to add custom extractors via api (#484) (Michael Ashley)
```

Comment:
Can you add the date/etc to the changelog to separate this release from the previous? (Just take a look at the existing structure of the changelog and duplicate that for these new release notes.)